extensions-core/caffeine-lrucache/runtime/pom.xml

```diff
@@ -52,39 +52,21 @@
 
     <build>
         <plugins>
-            <plugin>
-                <groupId>org.apache.camel.quarkus</groupId>
-                <artifactId>camel-quarkus-package-maven-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>update-extension-doc-page</id>
-                        <goals><goal>update-extension-doc-page</goal></goals>
-                        <phase>process-classes</phase>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>io.quarkus</groupId>
-                <artifactId>quarkus-bootstrap-maven-plugin</artifactId>
-            </plugin>

```

Comment:
Removing the dupicate quarkus-bootstrap-maven-plugin would be fine, but why are you removing also `camel-quarkus-package-maven-plugin`?

Reply:
next round

Reply:
got confused too, it is still there

Reply:
Indeed, sorry for the confusion.

Reply:
no worries